apps/js_urls/README.md

```diff
@@ -0,0 +1,36 @@
+# JavaScript URL Exporter
```

Comment:
- `js_routes` might sound better?
- Is there any reason this is a separate app under HTK and not just directly in HTK? I'm thinking that apps are better if they also have `models` or substantial amount of functionality.

Reply:
Removed as an `app` and added as a templatetag to `htk_tags`.
default constants moved to `htk.constants.defaults`